services/wikiService.js

```diff
@@ -0,0 +1,27 @@
+var request = require('request');
+var URL =  'https://en.wikipedia.org/w/api.php?format=json&action=query&generator=search&gsrnamespace=0&gsrlimit=4&prop=pageimages|extracts&pilimit=max&exintro&explaintext&exsentences=1&exlimit=max&gsrsearch=';
+module.exports = function (cb,username,input) {
+    'use strict';
+    let apiOptions = {
+        url :`${URL}${input}`,
+        method : 'GET',
+        json : {} //it is required
```

Comment:
I don't think the comment is needed here. Anyone reading this code down the line might get confused on reading this.